azure-devops/azext_devops/dev/boards/_format.py

```diff
@@ -114,3 +115,88 @@ def transform_work_item_query_result_row_output(row):
             # limit number of columns in table view
             break
     return table_row
+
+
+def transform_work_item_team_iterations_table_output(result):
+    table_output = []
+    for item in result:
```

Comment:
Minor: I think it might be useful to print it alphabetically (right now it is printed in the manner it comes from API)
we can use the key thing